edumfa/models.py

```diff
@@ -1632,14 +1632,15 @@ class PolicyCondition(MethodsMixin, db.Model):
     comparator = db.Column(db.Unicode(255), nullable=False, default='equals')
     Value = db.Column(db.Unicode(2000), nullable=False, default='')
     active = db.Column(db.Boolean, nullable=False, default=True)
+    undef = db.Column(db.Unicode(255), nullable=False, default=u'exception')
```

Comment:
you used *unset* above, better stick with one name

Reply:
Ahhh ... thats prop. also one reason for the failing tests. At least this must be done in a real database migration